cmake/OpenCVFindVA_INTEL.cmake

```diff
@@ -33,12 +19,12 @@ if(UNIX AND NOT ANDROID)
     DOC "Path to Intel OpenCL headers")
 endif()
 
-if(VA_INTEL_MSDK_INCLUDE_DIR AND VA_INTEL_IOCL_INCLUDE_DIR)
+if(VA_INTEL_IOCL_INCLUDE_DIR)
     set(HAVE_VA_INTEL TRUE)
     set(VA_INTEL_LIBRARIES "-lva" "-lva-drm")
```

Comment:
Similar to [this](https://github.com/opencv/opencv/blob/3.4.3/cmake/OpenCVFindVA.cmake#L15-L17) (allow to specify fulls path to libva custom builds):
```
    if(NOT DEFINED VA_INTEL_LIBRARIES)
      set(VA_INTEL_LIBRARIES "va" "va-drm")
    endif()
```